srcpkgs/libpurple-mattermost/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'libpurple-mattermost'
+pkgname=libpurple-mattermost
+_git_name=purple-mattermost
+version=1.2
+revision=1
+wrksrc="$_git_name-$version"
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="libpurple-devel json-glib-devel libglib-devel discount-devel base-devel"
```

Comment:
no need to have base-devel

Reply:
I thought `build-essential` which is in [the linux install guide](https://github.com/EionRobb/purple-mattermost/blob/master/INSTALL.md#linux-client-install-guide) was replaced in void by base-devel ?

Reply:
I get the answer on the irc channel, actually base-devel is already included into the builder no need to add it in makedepends